### PR TITLE
[Improvement] Added class to `tabBar`

### PR DESCRIPTION
**Pull request by Corepex**
Belongs To: https://github.com/pimcore/admin-ui-classic-bundle/pull/128

### CODE REVIEW

**Comment by blankse** [comment]
If the CSS class is only added in Pimcore 11.1, we should also increase the version to 11.1 in composer.json

**Comment by blankse** [comment]
If the CSS class is only added in pimcore/admin-ui-classic-bundle 1.1, we should also increase the version to 1.1 in composer.json

File: public/js/document/printcontainer.js [hunk 1]
```diff
@@ -65,18 +65,7 @@ pimcore.document.printcontainer = Class.create(pimcore.document.printabstract, {
...
-            region:'center',
-            deferredRender:true,
-            enableTabScroll:true,
-            defaults: {autoScroll:true},
-            border: false,
-            items: items,
-            activeTab: 0
-        });
-        return this.tabbar;
+        return pimcore.helpers.getTabBar({items: items, defaults: {autoScroll:true}});
```
Comment (robertSt7) [inline public/js/document/printcontainer.js]:
```suggestion
        this.tabbar = pimcore.helpers.getTabBar({items: items, defaults: {autoScroll:true}});
        return this.tabbar;
```

File: public/js/document/printpage.js [hunk 2]
```diff
@@ -75,19 +75,8 @@ pimcore.document.printpage = Class.create(pimcore.document.printabstract, {
...
-            region:'center',
-            deferredRender:true,
-            enableTabScroll:true,
-            defaults: {autoScroll:true},
-            border: false,
-            items: items,
-            activeTab: 0
-        });
-        return this.tabbar;
+        return pimcore.helpers.getTabBar({items: items, defaults: {autoScroll:true}});
```
Comment (robertSt7) [inline public/js/document/printpage.js]:
```suggestion
         this.tabbar = pimcore.helpers.getTabBar({items: items, defaults: {autoScroll:true}});
        return this.tabbar;
```

**Review by robertSt7 — CHANGES_REQUESTED** [review]

**Pull request merged**